packages/form-js-viewer/src/render/components/form-fields/Table.js

```diff
@@ -45,7 +45,9 @@ export function Table(props) {
   const evaluatedColumns = useEvaluatedColumns(columnsExpression || '', columns);
   const columnKeys = evaluatedColumns.map(({ key }) => key);
   const evaluatedDataSource = useExpressionEvaluation(dataSource);
-  const data = Array.isArray(evaluatedDataSource) ? evaluatedDataSource.filter((i) => i !== undefined) : [];
+  const data = Array.isArray(evaluatedDataSource)
```

Comment:
I couldn't really find a way to test this. Karma didn't throw any errors and it would render an empty table.

If you have any ideas let me know.